roles/st2chatops/vars/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+supported_hubot_adapters:
+ - slack
+ - hipchat
+ - xmpp
+ - flowdock
+ - yammer
+ - spark
+ - irc
+ - mattermost
```

Comment:
Just discovered via `st2chatops` PR that the right naming for mattermost adapter is `matteruser`, see: https://github.com/StackStorm/st2chatops/blob/1164c0823885931e3a3ad48e0cbb52a336f2d01b/st2chatops.env#L107